index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main-doc">
+      <nav id="navbar">
+        <header>Programming Languages</header>
+        <a href="#html" class="nav-link">HTML</a>
+        <a href="#css" class="nav-link">CSS</a>
+        <a href="#JavaScript" class="nav-link">JavaScript</a>
+        <a href="#python" class="nav-link">Python</a>
+        <a href="#java" class="nav-link">Java</a>
+      </nav>
+      <section class="main-section" id="html">
+        <header>HTML</header>
```

Comment:
The header in each section should have an h1-h6 element. This would give your code a better structure.